Anlab.Mvc/Client/components/Summary.tsx

```diff
@@ -56,6 +56,18 @@ export class Summary extends React.Component<ISummaryProps, any> {
         if (this.props.testItems.length === 0) {
             return null;
         }
+        let errorIconStyle = {
+            color: 'red',
+            padding: '0 10px'
+        }
+        let errorLink = null;
+        if (!this.props.hideError)
+        {
+            if (this.props.quantity < 1)
+                errorLink = <a href="#quantity">Fix Errors<i className="fa fa-exclamation" style={errorIconStyle} ></i></a>
+            else
```

Comment:
@laholstege let's not assume these will be the only two errors, so throw an IF into this else and do a `else if (!this.props.project)` so we know project is empty before showing that as the error